.github/workflows/ci-pull-request.yml

```diff
@@ -1,14 +1,16 @@
 name: ci-pull-request
 on:
   pull_request:
-    types: [opened, reopened, synchronize, closed]
+    branches:
+      - main
+      - master
+  push:
     branches:
       - main
       - master
 
 jobs:
   ci-pull-request:
-    if: ${{ github.event.action != 'closed' || github.event.pull_request.merged == true }}
```

Comment:
Does it have to run on all event types? Seems like we don't want a closed PR that isn't going to be merged, for example.

Reply:
Ah, I see. Thanks.